packages/desktop-client/src/components/budget/index.js

```diff
@@ -29,38 +36,45 @@ import { ReportProvider } from './report/ReportContext';
 import * as rollover from './rollover/rollover-components';
 import { RolloverContext } from './rollover/RolloverContext';
 
-let _initialBudgetMonth = null;
-
-class Budget extends PureComponent {
-  constructor(props) {
-    super(props);
-
-    const currentMonth = _initialBudgetMonth || monthUtils.currentMonth();
-    this.state = {
-      initialized: false,
-      prewarmStartMonth: props.startMonth || currentMonth,
-      newCategoryForGroup: null,
-      isAddingGroup: false,
-      collapsed: props.collapsedPrefs || [],
-      bounds: { start: currentMonth, end: currentMonth },
-      categoryGroups: null,
-      summaryCollapsed: props.summaryCollapsed,
-    };
-  }
+const currentMonth = monthUtils.currentMonth();
```

Comment:
Shouldn't this be inside the `Budget` function to have the same behavior as the class based component?

Reply:
Good catch: https://github.com/actualbudget/actual/pull/1566/commits/b2f4460b10825793fb525dc1235a5b913f3c3c98